[W17-2] OurTeeth

kthSim opened the pull request.

sharan8 commented on README.adoc:
```diff
@@ -14,13 +14,12 @@ ifndef::env-github[]
+* This is a desktop OurTeeth application. It has a GUI but most of the user interactions happen using a CLI (Command
+Line Interface).
+* Targeted towards Dentists who prefer to type.
+* It is developed in Java through the efforts of a five man team from NUS
```
Could use a fullstop for this sentence as well, for consistency

sharan8 commented on docs/AboutUs.adoc:
```diff
@@ -4,53 +4,60 @@
-=== John Doe
-image::damithc.jpg[width="150", align="left"]
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+image::kylerwsm.png[width="150", align="left"]
```
Could zoom in a little on this picture to be able to identify the team member better

sharan8 commented on docs/AboutUs.adoc:
```diff
@@ -4,53 +4,60 @@
-Role: Team Lead +
-Responsibilities: UI
+Role: Developer +
+Responsibilities: Records Managament
```
Do try to standardise the formatting of each member's section

sharan8 replied:
Do take note of the spelling of 'management' here

sharan8 commented on docs/AboutUs.adoc:
```diff
@@ -4,53 +4,60 @@
 
 Role: Developer +
-Responsibilities: Dev Ops + Threading
+Responsibilities: Login
```
Do update your responsibilities as you go along

sharan8 commented on src/main/java/seedu/address/MainApp.java:
```diff
@@ -174,6 +174,11 @@ public void start(Stage primaryStage) {
+        if (model.checkNoCopy()) {
+            logger.info("No copies left");
+        } else {
+            logger.info("Unedited copies exist");
```
Good that you are making use of logger messages

sharan8 commented on src/main/java/seedu/address/logic/commands/CopyCommand.java:
```diff
@@ -0,0 +1,62 @@
+import seedu.address.model.person.Person;
+
+/**
+ * Adds a person to the address book.
```
Remember to update your comments when reusing code

sharan8 commented on src/main/java/seedu/address/logic/commands/CopyCommand.java:
```diff
@@ -0,0 +1,62 @@
+    private final Index index;
+
+    /**
+     * Creates an AddCommand to add the specified {@code Person}
```
Comment to be updated

sharan8 commented on src/main/java/seedu/address/logic/commands/ExportCommand.java:
```diff
@@ -0,0 +1,63 @@
+            + "Example: " + COMMAND_WORD + " records1.json";
+
+    public static final String MESSAGE_SUCCESS = "Exported the records!";
+    private static final String MESSAGE_FAILURE = "Problem while writing to the file.";
```
Could look into why one of these constants is public while the other is kept private

sharan8 commented on src/main/java/seedu/address/logic/commands/ImportCommand.java:
```diff
@@ -0,0 +1,82 @@
+
+        StorageManager storage = new StorageManager(addressBookStorage, null);
+
+        final Logger logger = LogsCenter.getLogger(MainApp.class);
```
Moving forward, do think about whether this line would be best positioned inside or outside of this method, depending on where you intend to use the logger

sharan8 commented on src/main/java/seedu/address/logic/commands/ImportCommand.java:
```diff
@@ -0,0 +1,82 @@
+                logger.info("Data file not found. Will be starting with a sample AddressBook");
+            }
+            initialData = addressBookOptional.orElseGet(SampleDataUtil::getSampleAddressBook);
+        } catch (DataConversionException e) {
```
Good that you are handling the exception cases without crashing/exiting the application

sharan8 commented on src/main/java/seedu/address/logic/parser/ExportCommandParser.java:
```diff
@@ -0,0 +1,28 @@
+            throw new ParseException(pe.getMessage());
+        }
+    }
+    // TODO: Combine ImportCommandParser and ExportCommandParser
```
Would be good if the team could make use of the issue tracker to keep track of such TODO items to be implemented as well

sharan8 commented on src/main/java/seedu/address/model/ModelManager.java:
```diff
@@ -212,6 +212,11 @@ private void ensureSelectedPersonIsValid(ListChangeListener.Change<? extends Per
     }
 
+    @Override
+    public boolean checkNoCopy() {
```
Do refer to the coding style document on naming of boolean variables to identify a name that might be more appropriate

sharan8 commented on src/main/java/seedu/address/model/patient/Teeth.java:
```diff
@@ -0,0 +1,143 @@
+    private static final String NONE = "none";
+    private static final String CHILD = "child";
+    private static final String ADULT = "adult";
+    private static final int PRIMARYTEETHCOUNT = 20;
```
Do refer to the coding style on the naming of constants

sharan8 commented on src/main/java/seedu/address/model/patient/exceptions/PersonIsNotPatient.java:
```diff
@@ -0,0 +1,10 @@
+/**
+ * Signals the operation that a person is not a patient instance.
+ */
+public class PersonIsNotPatient extends RuntimeException {
```
A convention to follow when naming custom Exceptions would be to append 'Exception' to the end of the class name

sharan8 commented on src/main/java/seedu/address/model/person/Age.java:
```diff
@@ -0,0 +1,53 @@
+ */
+public class Age {
+
+
```
Take note of the extra line here

sharan8 commented on src/main/java/seedu/address/model/person/DrugAllergy.java:
```diff
@@ -0,0 +1,60 @@
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    //TODO: Discuss if full drug name or use short form instead
```
Try to omit such to-dos from the source code

sharan8 commented on src/test/java/seedu/address/model/person/PersonTest.java:
```diff
@@ -35,13 +35,13 @@ public void isSamePerson() {
         assertFalse(ALICE.isSamePerson(editedAlice));
 
-        // different name -> returns false
+        // different name -> returns true
```
Curious as to what is the intention behind this allowance

sharan8 commented on src/test/java/systemtests/EditCommandSystemTest.java:
```diff
@@ -49,18 +51,20 @@
 
     @Test
-    public void edit() {
+    public void edit1() {
```
Do try to use more descriptive test method names where possible

sharan8 reviewed (commented):
Hi team, some comments:

- Do strive for more consistency in formatting across similar sections in your docs.
- Do remember to update your responsibilities in the AboutUs page, and be as precise as possible.
- Remember to update your comments/javadoc when reusing code from elsewhere in the project. But good that there is an effort to add comments/javadocs for methods in the new classes.
- Do refer to the recommended coding style with regards to the naming of boolean variables and custom exceptions.
- Look out for some stray extra lines in the code.
- Just re-iterating the single user constraint for the project -> Might want to give the Login feature a deeper thought.

Comments on your User and Developer Guides will be provided separately (likely as annotated PDFs on Luminus).

Overall, good that the groundwork for the project has been laid, and do keep up the work while updating your docs/UI image as you go along. Where cross-component interaction is necessary, do remember to discuss and align yourselves when implementing your respective features.

sharan8 commented:
Update on documentation feedback:

- Some feedback will be provided for the User Guide, however do furnish your Developer Guide with your user stories, use cases and NFRs. It doesn't seem to be updated fully yet.
- In the meantime, you can work on the comments provided for your past Developer Guide submission.

sharan8 commented on docs/DeveloperGuide.adoc:
```diff
@@ -827,15 +827,45 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+|`* * *` |user |add a new patient's particulars | know about their situation
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |edit my patient's particulars when the situation changes |
```
Updating 'when the situation changes' might be more appropriate under 'So that I can...'

sharan8 commented on src/main/java/seedu/address/logic/commands/ExitCommand.java:
```diff
@@ -10,11 +10,19 @@
-    public static final String MESSAGE_EXIT_ACKNOWLEDGEMENT = "Exiting Address Book as requested ...";
+    static final String MESSAGE_EXIT_ACKNOWLEDGEMENT = "Exiting Address Book as requested ...";
+
+    private static final String MEESAGE_COPY_EXIST = "!eExists unedited copies in addressbook.\n"
```
MESSAGE*

sharan8 commented on src/main/java/seedu/address/logic/commands/TaskAddCommand.java:
```diff
@@ -0,0 +1,65 @@
+
+
+/**
+ * Adds a patient to the address book.
```
*task

sharan8 reviewed (commented):
Hi team, some comments:

- Do fix your checkstyle issues in a timely manner to avoid the build failures.
- As mentioned in the demo last tutorial, do try to carry out some background research to better cater your application to your intended target audience. Hope the group has worked out the feature allocations as well.
- For your features, do think deeper into how they add value to your intended user. It may be helpful to outline and contextualize the intended purpose behind the features in your User Guide as well. (E.g. How can the copy feature be used to enhance productivity for the intended user?)
- Do remember to update your `[Coming in v...]` tags accordingly. If some features aren't fully implemented yet, do mention which aspects are yet to be implemented in future releases.
- Moving forward, do think about how diagrams, tables, etc. could be better used to enhance the delivery of information in your Guides. It may also be helpful to reorganise some of the more wordy content in a more digestible manner.
- You're encouraged to update the diagrams in your Developer Guide to reflect the changes made.
- Descriptions of implementation appear to be absent for your features. Refer to the [v1.2 Documentation Requirements](https://nus-cs2103-ay1819s2.github.io/cs2103-website/admin/project-w09-v12.html).
- Do add your Use Cases to the Developer Guide as well. I would suggest that 1/2 members of the team take charge of ensuring that the documentation is in order prior to every release.
- Do think about how you can apply certain topics discussed in class to refactor your code, if you feel that you have adequately implemented your feature(s).
- Looking forward to reading about why your features are implemented in the current manner, and what were the alternatives considered.

kylerwsm committed:
Updated UG

kylerwsm committed:
Updated UG

kylerwsm committed:
Updated UG

cchj1995 committed:
Update DeveloperGuide.adoc

kylerwsm committed:
Updated UG

cchj1995 committed:
Updated TaskDoneSequenceDiagram

cchj1995 committed:
Updated DG

cchj1995 committed:
Shifted images up

kylerwsm committed:
Updated PPP, AboutUs

cchj1995 committed:
Update DeveloperGuide.adoc

kylerwsm committed:
Updated PPP writeup

cchj1995 committed:
Update DeveloperGuide.adoc

kylerwsm committed:
Updated PPP

kthSim committed:
Resolves #161, updates user Guide

kylerwsm committed:
Updated DG part in PPP

kthSim committed:
Shift order of image

kthSim committed:
update preamble

kylerwsm committed:
Edited PPP

kylerwsm committed:
Updated DG

kylerwsm committed:
Merge branch 'portfolio' of ssh://github.com/kylerwsm/main into portfolio

kthSim committed:
added umbrella

kylerwsm committed:
Updated PPP

kylerwsm committed:
Removed the dentist command

kylerwsm committed:
Draft1 of PPP

kthSim committed:
minor formatting

kylerwsm committed:
Updated PPP and UG

cchj1995 committed:
Update DeveloperGuide.adoc

JiaHaoLim committed:
Update DeveloperGuide.adoc

kylerwsm committed:
Updated PPP and UG

kylerwsm committed:
Merge branch 'documentation' into portfolio

kylerwsm committed:
Updated PPP and UG

kylerwsm committed:
Updated PPP and UG

cchj1995 committed:
Update DeveloperGuide.adoc

kthSim committed:
Merge pull request #196 from kthSim/docs

[v1.4.1] Updated documentation

JiaHaoLim committed:
Update DeveloperGuide.adoc

Refactored File Management and Suggestion implementations.

JiaHaoLim committed:
Update DeveloperGuide.adoc

JiaHaoLim committed:
Test

JiaHaoLim committed:
Update DeveloperGuide.adoc

kthSim committed:
Merge pull request #198 from JiaHaoLim/update-documentation

[v1.4] Update documentation

cchj1995 committed:
Update DeveloperGuide.adoc

kthSim committed:
updated find dev docs

cchj1995 committed:
Update DeveloperGuide.adoc

kthSim committed:
updated design consideration of sort command

kthSim committed:
format update

JiaHaoLim committed:
Added Personal Project Portfolio jiahaolim.adoc.

JiaHaoLim committed:
Updated jiahaolim.adoc.

cchj1995 committed:
Update DeveloperGuide.adoc

cchj1995 committed:
Update DeveloperGuide.adoc

cchj1995 committed:
Update DeveloperGuide.adoc

kthSim committed:
updated PP

kthSim committed:
udpated tagging

kthSim committed:
updated dev gudie contents for stat

cchj1995 committed:
Update UserGuide.adoc

JiaHaoLim committed:
Update DeveloperGuide.adoc

Updated Find Feature Implementation's Design Consideration to conform to the new format.

cchj1995 committed:
Added new image

cchj1995 committed:
Update UserGuide.adoc

cchj1995 committed:
Update UserGuide.adoc

cchj1995 committed:
Update UserGuide.adoc

JiaHaoLim committed:
Update DeveloperGuide.adoc

JiaHaoLim committed:
Updated jiahaolim.adoc.

JiaHaoLim committed:
Merge branch 'update-documentation' of https://github.com/JiaHaoLim/main into update-documentation

JiaHaoLim committed:
Updated jiahaolim.adoc.

JiaHaoLim committed:
Changed File.separator back to "/" as it broke when compiled to a .jar file.

JiaHaoLim committed:
Merge branch 'common-IO' into add-tests

JiaHaoLim committed:
Changed File.separator back to "/" as it broke when compiled to a .jar file.

JiaHaoLim committed:
Changed File.separator back to "/" as it broke when compiled to a .jar file.

JiaHaoLim committed:
Merge branch 'common-IO'

JiaHaoLim committed:
Merge branch 'add-tests'

JiaHaoLim committed:
Removed unused imports.

JiaHaoLim committed:
Updated ParserUtilTest.java

JiaHaoLim committed:
Merge branch 'common-IO'

JiaHaoLim committed:
Merge branch 'add-tests'

JiaHaoLim committed:
Removed trailing whitespace.

kthSim committed:
updated PPP

kthSim committed:
Merge branch 'documentation' into portfolio

kthSim committed:
removed prefix link in patientedit

kthSim committed:
Merge pull request #201 from kthSim/docs

[v1.4.2] Documenation & PPP updates

kthSim committed:
Merge pull request #199 from JiaHaoLim/update-documentation

Update documentation

kthSim committed:
Merge pull request #200 from JiaHaoLim/master

[v1.4] Bugfix

kthSim committed:
Merge pull request #197 from kylerwsm/portfolio

[v1.4.5] PPP Draft 1 and Updates to UG and DG

kthSim committed:
Merge branch 'master' into documentation

kthSim committed:
Merge pull request #202 from CS2103-AY1819S2-W17-2/documentation

Documentation

kthSim committed:
temp remove

kthSim committed:
update Ui

JiaHaoLim committed:
Updated AboutUs.adoc

cchj1995 committed:
Update UserGuide.adoc

cchj1995 committed:
Minor changes

cchj1995 committed:
Added images

cchj1995 committed:
Updated image

cchj1995 committed:
Updated image

cchj1995 committed:
Update UserGuide.adoc

cchj1995 committed:
Minor changes

cchj1995 committed:
Merge branch 'Documentation'

# Conflicts:
#	docs/images/Taskdone.png
#	preferences.json

cchj1995 committed:
Merged documentation

cchj1995 committed:
Merge branch 'master' of https://github.com/cchj1995/addressbook-level4

cchj1995 committed:
Minor changes

cchj1995 committed:
Updated image

cchj1995 committed:
Removed image

cchj1995 committed:
Re-added image

cchj1995 committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W17-2/main

# Conflicts:
#	docs/DeveloperGuide.adoc

kylerwsm committed:
Updated PPP and UG

kylerwsm committed:
Updated PPP and UG

JiaHaoLim committed:
Trimmed jiahaolim.adoc

JiaHaoLim committed:
Updated jiahaolim.adoc

JiaHaoLim committed:
Added newline at the end of jiahaolim.adoc

cchj1995 committed:
Update UserGuide.adoc

cchj1995 committed:
Update cchj1995.adoc

kylerwsm committed:
Updated PPP

kylerwsm committed:
Updated PPP

kthSim committed:
Merge pull request #205 from JiaHaoLim/update-documentation

[v1.4] PPP update

kylerwsm committed:
Updated PPP

kylerwsm committed:
Updated PPP

kylerwsm committed:
Updated PPP

kylerwsm committed:
Updated PPP

kthSim committed:
Update kylerwsm.adoc

kthSim committed:
Merge pull request #204 from kylerwsm/master

[v1.4.6] PPP Draft 2

kthSim committed:
PPP updates

kthSim committed:
Merge pull request #206 from kthSim/master

PPP update

kylerwsm committed:
Updated with rectangular photo of self

kylerwsm committed:
Merge branch 'master' of ssh://github.com/kylerwsm/main

JiaHaoLim committed:
Changed jiahaolim.png back to the rectangular version.

kylerwsm committed:
Fixed the filepath problem for TeethHub.json

kthSim committed:
udpated PPP data

JiaHaoLim committed:
Changed backward slash to forward slash in preferences.json.

kylerwsm committed:
Added new line at EOF

cchj1995 committed:
Update cchj1995.adoc

cchj1995 committed:
Update cchj1995.adoc

cchj1995 committed:
Update AboutUs.adoc

cchj1995 committed:
Update cchj1995.adoc

cchj1995 committed:
Changed shadowjar name

JiaHaoLim committed:
Merge branch 'common-IO'

cchj1995 committed:
Update UserGuide.adoc

cchj1995 committed:
Update cchj1995.adoc

cchj1995 committed:
Minor changes

cchj1995 committed:
Update cchj1995.adoc

cchj1995 committed:
Test

JiaHaoLim committed:
Updated Open Command Sequence Diagram.

cchj1995 committed:
Minor changes

JiaHaoLim committed:
Merge branch 'update-documentation'

cchj1995 committed:
Updated PPP

cchj1995 committed:
Update cchj1995.adoc

cchj1995 committed:
Update cchj1995.adoc

cchj1995 committed:
Update DeveloperGuide.adoc

cchj1995 committed:
Update DeveloperGuide.adoc

cchj1995 committed:
Update cchj1995.adoc

cchj1995 committed:
Fixed some checkstyle errors

cchj1995 committed:
Fixed hopefully

cchj1995 committed:
Style change

cchj1995 committed:
Ok this should fix it

cchj1995 committed:
Apparently not

JiaHaoLim committed:
Update jiahaolim.adoc

JiaHaoLim committed:
Updated README.adoc and jiahaolim.adoc.

kthSim committed:
Merge pull request #207 from kylerwsm/master

[v1.4.7] Documentation and File Path Fix

kthSim committed:
Merge pull request #210 from cchj1995/master

Updates for DG, UG and PPP

kthSim committed:
Merge pull request #208 from JiaHaoLim/master

[v1.4] Minor change in Preferences.json

JiaHaoLim committed:
Removed trailing whitespace.

kthSim committed:
Merge pull request #209 from JiaHaoLim/update-documentation

[v1.4] Update Documentation

kthSim committed:
PPP updates

cchj1995 committed:
Updated Documentation to fix PPP format in PDF mode

cchj1995 committed:
Updated PPP and DG

kthSim committed:
Merge pull request #211 from kthSim/master

PP update

kthSim committed:
Merge pull request #212 from cchj1995/master

Updated PPP

cchj1995 committed:
Major changes

cchj1995 committed:
Minor changes

cchj1995 committed:
Fixed errors in UserGuide

cchj1995 committed:
Merge branch 'master' into master

wSemis committed:
Red format for Unknown command

wSemis committed:
Print mc to a pdf file

wSemis committed:
Bug fix for creating mass copies

wSemis committed:
Add RecordMcCommand class

wSemis committed:
Add RecordMcCommandParser class

wSemis committed:
Add RecordMc command to the parser

wSemis committed:
Parse function for RecordMc

wSemis committed:
Merge branch 'master' of https://github.com/wSemis/main

wSemis committed:
Merge https://github.com/wSemis/main

# Conflicts:
#	src/test/java/systemtests/PatientClearCommandSystemTest.java
#	src/test/java/systemtests/PatientDeleteCommandSystemTest.java
#	src/test/java/systemtests/PatientSelectCommandSystemTest.java

wSemis committed:
Merge https://github.com/wSemis/main

# Conflicts:
#	src/test/java/systemtests/PatientClearCommandSystemTest.java
#	src/test/java/systemtests/PatientDeleteCommandSystemTest.java
#	src/test/java/systemtests/PatientSelectCommandSystemTest.java

wSemis committed:
Merge https://github.com/wSemis/main

# Conflicts:
#	src/test/java/systemtests/ModelHelper.java

wSemis committed:
Merge https://github.com/wSemis/main

# Conflicts:
#	src/test/java/systemtests/ModelHelper.java

wSemis committed:
Style change

kthSim committed:
Merge pull request #213 from cchj1995/master

Updated UserGuide, PPP, SampleTaskData. Removed unused code

wSemis committed:
Add test cases

wSemis committed:
Bug fix

wSemis committed:
Merge

wSemis committed:
Bug fix

wSemis committed:
-

wSemis committed:
Style change

wSemis committed:
Style change

wSemis committed:
Style change

wSemis committed:
Test case bug fix

wSemis committed:
bug fix

wSemis committed:
Update Messages.java

kthSim committed:
Merge pull request #214 from wSemis/master

V1.4 RecordMcCommand, Testcases, Patientcopy bug fix

kthSim committed:
Update DeveloperGuide.adoc

kthSim committed:
Update DeveloperGuide.adoc

kthSim committed:
removed old window

kthSim committed:
added new stat

kthSim committed:
Merge pull request #215 from kthSim/master

minor image renaming

JiaHaoLim committed:
Updated jiahaolim.adoc

JiaHaoLim committed:
Merge branch 'update-documentation'

JiaHaoLim committed:
Merge pull request #21 from CS2103-AY1819S2-W17-2/master

Update forked repo

JiaHaoLim committed:
Merge pull request #22 from CS2103-AY1819S2-W17-2/documentation

Documentation

JiaHaoLim committed:
Merge branch 'update-documentation'

JiaHaoLim committed:
Minor changes to wordings.

JiaHaoLim committed:
Added new Documentation contributions under "Other Contributions" instead of including them.

JiaHaoLim committed:
Added new entry in User Guide.

JiaHaoLim committed:
Added new test cases for Manual Testing in Developer Guide.

kylerwsm committed:
Updated UG, DG, PPP

kthSim committed:
Merge pull request #216 from JiaHaoLim/master

[v1.4] Documentation changes.

kthSim committed:
Merge pull request #217 from kylerwsm/master

[v1.4.8] Updated UG, DG, PPP

cchj1995 committed:
Updated UserGuide, changed display messages

cchj1995 committed:
Fixed style, removed unused content from Developer's Guide

cchj1995 committed:
Fixed style, removed unused content from Developer's Guide

cchj1995 committed:
Added newline to EOF in preferences.json

wSemis committed:
Update documents

wSemis committed:
Style change

kthSim committed:
Merge pull request #218 from cchj1995/master

Updated DG, Fixed UG, Modified Task feature

wSemis committed:
Merge branch 'master' into master

wSemis committed:
Merge branch 'master' into master

wSemis committed:
Update UserGuide.adoc

wSemis committed:
Style change

kthSim committed:
Update UserGuide.adoc

wSemis committed:
Change to asce/desc

JiaHaoLim committed:
Update UserGuide.adoc

JiaHaoLim committed:
Update jiahaolim.adoc

kthSim committed:
Merge pull request #220 from JiaHaoLim/patch-1

Update UserGuide.adoc

kthSim committed:
Merge pull request #219 from wSemis/master

V1.4 Documents added for Debang

kthSim committed:
Merge pull request #221 from JiaHaoLim/patch-2

Update jiahaolim.adoc

kthSim committed:
Update AboutUs.adoc

wSemis committed:
Update AboutUs.adoc

wSemis committed:
Update AboutUs.adoc

kthSim committed:
Merge pull request #222 from wSemis/master

Update AboutUs.adoc

wSemis committed:
Typo change

wSemis committed:
Merge branch 'master' of https://github.com/wSemis/main

kthSim committed:
Update UserGuide.adoc

kthSim committed:
Update UserGuide.adoc

kthSim committed:
Update UserGuide.adoc

kthSim committed:
Update UserGuide.adoc

kthSim committed:
Merge pull request #223 from wSemis/master

Update documents

JiaHaoLim committed:
Readded changes that was removed in some Pull Request merge.

JiaHaoLim committed:
Merge branch 'update-documentation'

kthSim committed:
Update AboutUs.adoc

cchj1995 committed:
Fixed bugs in code and UG, updated messages

wSemis committed:
Update AboutUs.adoc

kthSim committed:
Update AboutUs.adoc

kthSim committed:
Merge branch 'master' into master

cchj1995 committed:
Updates

kthSim committed:
Merge pull request #225 from wSemis/master

Update AboutUs.adoc

kthSim committed:
Merge pull request #226 from cchj1995/master

Fixed bugs in code and UG, updated messages

kthSim committed:
Merge branch 'master' into master

kthSim committed:
Merge pull request #224 from JiaHaoLim/master

 Readded changes that was removed in some Pull Request merge.
